src/compress.egg-info/top_level.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
You can include this cleanup in a final polish step towards the end of the work. Ditto for src/compress.egg-info/dependency_links.txt.